pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -583,9 +585,43 @@ self: super:
         propagatedBuildInputs = [ libpciaccess epoxy ] ++ commonPropagatedBuildInputs ++ lib.optionals stdenv.isLinux [
           udev
         ];
-        prePatch = stdenv.lib.optionalString stdenv.hostPlatform.isMusl ''
+        prePatch = (attrs.prePatch or "") + stdenv.lib.optionalString stdenv.hostPlatform.isMusl ''
           export CFLAGS+=" -D__uid_t=uid_t -D__gid_t=gid_t"
         '';
+        patches = [
+          # NVIDIA PRIME Render Offload patchset
+          # Should be upstream by 1.21
+          # http://download.nvidia.com/XFree86/Linux-x86_64/435.17/README/primerenderoffload.html
+          (fetchpatch {
+            url = "https://gitlab.freedesktop.org/xorg/xserver/commit/7f962c70b6d9c346477f23f6c15211e749110078.patch";
+            sha256 = "196nzhw4wipz6a57j1k6wrp4j80z1fq4fpqhw3xcc1mb3yy24pf6";
```

Comment:
hash mismatch in fixed-output derivation '/nix/store/0b213zw9h2q283jn3cvpi18qgz00mj6h-7f962c70b6d9c346477f23f6c15211e749110078.patch':
  wanted: sha256:196nzhw4wipz6a57j1k6wrp4j80z1fq4fpqhw3xcc1mb3yy24pf6
  got:    sha256:0qv844fj7hbwq9mvcdxra2a4g1sl45lz614w7dc7h0hvk17lgcf0